TMDbLib/Client/TMDbClientMovies.cs

```diff
@@ -101,7 +101,7 @@ public partial class TMDbClient
             RestResponse<Movie> response = await req.ExecuteGet<Movie>(cancellationToken).ConfigureAwait(false);
 
             // No data to patch up so return
-            if (response == null) return null;
+            if (response == null || !response.IsValid) return null;
```

Comment:
We shouldn't check for IsValid in _some_ places.

Reply:
This is also related to trying to return null.. Response isn't null, as it's a wrapper containing the null value.

Reply:
Another thing to fix then is that if ExecuteGet always returns non-null - we should never check for null.

Reply:
Yeah, that's probably a good idea.